tests/testdata/npm/deno_run_non_existent.out

```diff
@@ -1,3 +1,2 @@
 Download http://localhost:4260/mkdirp
-Download http://localhost:4260/mkdirp
```

Comment:
It downloading this multiple times was a cosmetic bug previously from https://github.com/denoland/deno/pull/18622/. We already got the latest information from the npm registry, so we don't need to download it again (this is now enforced by the `RegistryInfoDownloader`)